### ISSUE TITLE
Powered by Tor?

### ISSUE THREAD

**Issue by Darth-Ness** [negyio/.github#1]
Is Negy powered by Tor, or just follows the Tor protocol because I thought Negy was it's own network (I mean requests go though Negy not Tor)? I'm asking because Tor has some questionable privacy issues [https://restoreprivacy.com/tor/](url)

**Comment by tbrand** [comment]
Yeah, you are right. I'll remove the phrase, thanks for the feedback! Sometimes I use the wrong phrase since I'm not a native. I appreciate if you fix my English :)
